library/std/src/sys/unix/fd.rs

```diff
@@ -266,22 +266,9 @@ impl FileDesc {
         }
     }
 
+    #[inline]
     pub fn duplicate(&self) -> io::Result<FileDesc> {
```

Comment:
Long-term, I wonder if we can eliminate this internal interface in favor of try_clone. But that's not an issue for this PR.